src/components/m-header-mobile-languages/js/header-mobile-languages.js

```diff
@@ -0,0 +1,52 @@
+import on from '../../../js/on';
+import { publish } from '../../../js/pubsub';
+
+class HeaderMobileLangugaes {
+  static DEFAULTS = {
+    link: 'js-header-mobile-languages__link',
+  }
+
+  constructor(rootNode, options) {
+    this.rootNode = rootNode;
+    this.options = {
+      ...HeaderMobileLangugaes.DEFAULTS,
+      ...options,
+    };
+
+    this.handleClick = this.handleClick.bind(this);
+
+    this.init();
+  }
+
+  set contextNode(value) {
```

Comment:
i would prefer to have setters and getter at the bottom of every class (java like). just a personal preference thought so can be ignored